pom.xml

```diff
@@ -205,7 +205,7 @@
             <dependency>
                 <groupId>org.apache.tinkerpop</groupId>
                 <artifactId>gremlin-groovy-test</artifactId>
-                <version>${tinkerpop.version}</version>
+                <version>3.2.11</version>
```

Comment:
keep ${tinkerpop.version}

Reply:
gremlin-groovy-test the newest version is 3.2.11